src/models/user_account.ts

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
Question: do we also need to update the GraphQL schema?

Reply:
Not yet, since we're not sending the user account over the wire since for the time being it doesn't contain anything the client doesn't already get directly from auth0